runtime/include/cloe/data_broker.hpp

```diff
@@ -748,8 +749,15 @@ std::size_t Container<TValue>::subscriber_count() const {
   */
 class DataBroker {
  private:
+  struct CompareName {
+    using is_transparent = void;
+    bool operator()(std::string const& lhs, std::string const& rhs) const { return lhs < rhs; }
+    bool operator()(std::string const& lhs, std::string_view const& rhs) const { return lhs < rhs; }
+    bool operator()(std::string_view const& lhs, std::string const& rhs) const { return lhs < rhs; }
+  };
+
   using signal_ptr_t = Signal::signal_ptr_t;
-  using signal_container_t = std::map<std::string, signal_ptr_t>;
+  using signal_container_t = std::map<std::string, signal_ptr_t, CompareName>;

```

Comment:
I think instead of `CompareName`, you can also use the opaque comparison operator `std::less<>`

Reply:
Done.